core/src/CopyChip/CopyChip.js

```diff
@@ -0,0 +1,87 @@
+import React, { useState, useEffect } from "react";
+import cx from "classnames";
+
+import { FontAwesomeIcon } from "@fortawesome/react-fontawesome";
+import { faClipboard, faCheck } from "@fortawesome/free-solid-svg-icons";
+
+import { Button } from "../Button";
+
+import styles from "./CopyChip.less";
+
+export const CopyChip = (props) => {
+  const defaultCopySuccessMessage = "Copied";
+  const { copySuccessMessage = defaultCopySuccessMessage, value } = props;
+
+  const [showCopySuccess, setCopySuccess] = useState(false);
```

Comment:
Stick with react conventions on the reference variable mapping to the set function. e.g. `copySuccess`, `setCopySuccess`

Reply:
also if there is "succes" then there can be a "failure". with this code that's not the case. we can go with simpler naming. e.g. `[copied, setCopied]`

then code using that value would read as `if (copied)`